### PR TITLE
chore(*): update CODEOWNERS

### PR DESCRIPTION
**Pull request by Salakar**
Updated as per https://github.com/firebase/extensions/pull/98#issuecomment-551266401

cc @rachelsaunders

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # All documentation changes, e.g. changelogs & readmes.
-*.md             @rachelsaunders
+*.md             @rachelsaunders @annie-ryan
```
Comment (laurenzlong) [inline .github/CODEOWNERS]: I'm not sure Annie needs to be in the CODEOWNERS list, she should no be expected to review things via Github. We will surface UX concerns during our Tuesday cross-functional syncs.

Comment (laurenzlong) [reply .github/CODEOWNERS]: Actually I just saw the comment @rachelsaunders left on the other PR. Rachel, let's discuss this with Annie offline.

Comment (rachelsaunders) [reply .github/CODEOWNERS]: I think that at minimum for this PR, we should have me be a "codeowner" for the the .md files and the extension.yaml files.  We can review the addition of Annie in the RACI review?

Comment (laurenzlong) [reply .github/CODEOWNERS]: That sounds good to me. @Salakar can you revise this PR to be just for adding Rachel as an owner of extension.yaml files, and we can revisit adding Annie in a subsequent PR if needed.

Comment (laurenzlong) [reply .github/CODEOWNERS]: Also, when I first wrote this comment, I accidentally edited Rachel's (that's scary that Github lets me edit other people's comments). I reverted Rachel's comment back to what it was before.

**Pull request merged**